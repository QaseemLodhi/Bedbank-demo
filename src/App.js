import React, { useState, user } from "react";
import { Formik } from "formik";

//import logo from "./logo.svg";
import {
  StyleSheet,
  Text,
  View,
  FlatList,
  TouchableOpacity,
  Alert,
  Keyboard,
  TextInput
} from "react-native";
import Calendar from "react-calendar";

import "./App.css";

const Header = () => (
  <View
    style={{
      backgroundColor: "#006ead",
      height: 100,
      flexDirection: "row",
      justifyContent: "space-between"
    }}
  >
    <View
      style={{
        alignItems: "center",
        flexDirection: "row"
        // backgroundColor: "blue"
      }}
    >
      <Text>SastaTicket.pk</Text>
      <Text
        style={{
          fontWeight: "bold",
          fontSize: 20,
          marginLeft: 10
        }}
      >
        Flights
      </Text>
      <Text
        style={{
          fontWeight: "bold",
          fontSize: 20,
          marginLeft: 10
        }}
      >
        Hotels
      </Text>
      <Text
        style={{
          fontWeight: "bold",
          fontSize: 20,
          marginLeft: 10
        }}
      >
        Deals
      </Text>
    </View>

    <View
      style={{
        //backgroundColor: "orange",
        flexDirection: "row",
        alignItems: "center"
      }}
    >
      <Text style={{ marginHorizontal: 5 }}>Login</Text>

      <Text style={{ marginHorizontal: 5 }}>Contact us</Text>
    </View>
  </View>
);

const Body = props => (
  <View
    style={{
      backgroundColor: "#006ead",
      height: 500,
      justifyContent: "center",
      alignItems: "center",
      marginTop: 10
    }}
  >
    {props.children}
  </View>
);

const Footer = () => (
  <View
    style={{
      backgroundColor: "#006ead",
      height: 200,
      justifyContent: "center",
      alignItems: "center",
      marginTop: 10
    }}
  >
    <Text
      style={{
        fontWeight: "bold",
        fontSize: 20
      }}
    >
      This Footer
    </Text>
  </View>
);

function fireTestEvent() {
  window.dataLayer &&
    window.dataLayer.push({
      event: "pagerefresh"
    });
}
function App() {
  const [count, setCount] = useState(0);
  fireTestEvent();
  return (
    <View style={styles.box}>
      {/* <View style={{ flex: 1, backgroundColor: "green" }} /> */}

      <Header />
      <Body>
        <Text>This is calender</Text>
        <Text>Some change to invalidate snapshot</Text>
        <View>
          <Text>Counter {count}</Text>
          <TouchableOpacity onPress={() => setCount(count + 1)}>
            <Text>Click me</Text>
          </TouchableOpacity>
        </View>
        <Calendar />
        <div className="App-link">
          <Text>Hello textasfdsa</Text>
        </div>
        <Formik
          initialValues={{ firstName: "" }}
          onSubmit={values => {
            Alert.alert(JSON.stringify(values, null, 2));
            Keyboard.dismiss();
          }}
        >
          {({ handleChange, handleSubmit, values }) => (
            <View style={{ backgroundColor: "white", marginTop: 5 }}>
              <TextInput
                onChangeText={handleChange("firstName")}
                value={values.firstName}
                label="First name"
                placeholder="I am ready!"
              />

              <TouchableOpacity
                onPress={handleSubmit}
                style={{ backgroundColor: "red", marginTop: 5 }}
              >
                <Text>Submit</Text>
              </TouchableOpacity>
            </View>
          )}
        </Formik>
      </Body>
      <Footer />

      {/* <FlatList
        data={[{ key: "a" }, { key: "b" }]}
        renderItem={({ item }) => (
          <View>
            <Text>{item.key}</Text>
          </View>
        )}
      />
      <Calendar /> */}
    </View>
  );
}

const styles = StyleSheet.create({
  box: {
    flex: 1,
    padding: 10,
    //backgroundColor: "red",
    flexDirection: "column"
    //height: "100%"
  },

  text: { fontWeight: "bold" }
});

export default App;
