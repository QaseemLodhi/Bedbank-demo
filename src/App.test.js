import React from "react";
import ReactDOM from "react-dom";
import App from "./App";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  shallow(<App />);
});

test("renders correctly", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
