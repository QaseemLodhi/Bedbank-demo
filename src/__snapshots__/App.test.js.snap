// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-flexDirection-eqz5dr r-padding-1sp51qo"
>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "horizontal",
        "WebkitBoxPack": "justify",
        "WebkitFlexDirection": "row",
        "WebkitJustifyContent": "space-between",
        "backgroundColor": "rgba(0,110,173,1.00)",
        "flexDirection": "row",
        "height": "100px",
        "justifyContent": "space-between",
        "msFlexDirection": "row",
        "msFlexPack": "justify",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": "center",
          "WebkitBoxAlign": "center",
          "WebkitBoxDirection": "normal",
          "WebkitBoxOrient": "horizontal",
          "WebkitFlexDirection": "row",
          "alignItems": "center",
          "flexDirection": "row",
          "msFlexAlign": "center",
          "msFlexDirection": "row",
        }
      }
    >
      <div
        className="css-text-901oao"
        dir="auto"
      >
        SastaTicket.pk
      </div>
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "fontSize": "20px",
            "fontWeight": "bold",
            "marginLeft": "10px",
          }
        }
      >
        Flights
      </div>
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "fontSize": "20px",
            "fontWeight": "bold",
            "marginLeft": "10px",
          }
        }
      >
        Hotels
      </div>
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "fontSize": "20px",
            "fontWeight": "bold",
            "marginLeft": "10px",
          }
        }
      >
        Deals
      </div>
    </div>
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": "center",
          "WebkitBoxAlign": "center",
          "WebkitBoxDirection": "normal",
          "WebkitBoxOrient": "horizontal",
          "WebkitFlexDirection": "row",
          "alignItems": "center",
          "flexDirection": "row",
          "msFlexAlign": "center",
          "msFlexDirection": "row",
        }
      }
    >
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "marginLeft": "5px",
            "marginRight": "5px",
          }
        }
      >
        Login
      </div>
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "marginLeft": "5px",
            "marginRight": "5px",
          }
        }
      >
        Contact us
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignItems": "center",
        "WebkitBoxAlign": "center",
        "WebkitBoxPack": "center",
        "WebkitJustifyContent": "center",
        "alignItems": "center",
        "backgroundColor": "rgba(0,110,173,1.00)",
        "height": "500px",
        "justifyContent": "center",
        "marginTop": "10px",
        "msFlexAlign": "center",
        "msFlexPack": "center",
      }
    }
  >
    <div
      className="css-text-901oao"
      dir="auto"
    >
      This is calender
    </div>
    <div
      className="css-text-901oao"
      dir="auto"
    >
      Some change to invalidate snapshot
    </div>
    <div
      className="css-view-1dbjc4n"
    >
      <div
        className="css-text-901oao"
        dir="auto"
      >
        Counter 
        0
      </div>
      <div
        className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        data-focusable={true}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        tabIndex="0"
      >
        <div
          className="css-text-901oao"
          dir="auto"
        >
          Click me
        </div>
      </div>
    </div>
    <div
      className="react-calendar"
      onBlur={null}
      onMouseOut={null}
    >
      <div
        className="react-calendar__navigation"
        style={
          Object {
            "display": "flex",
          }
        }
      >
        <button
          aria-label=""
          className="react-calendar__navigation__arrow react-calendar__navigation__prev2-button"
          onClick={[Function]}
          type="button"
        >
          «
        </button>
        <button
          aria-label=""
          className="react-calendar__navigation__arrow react-calendar__navigation__prev-button"
          onClick={[Function]}
          type="button"
        >
          ‹
        </button>
        <button
          aria-label=""
          className="react-calendar__navigation__label"
          disabled={false}
          onClick={[Function]}
          style={
            Object {
              "flexGrow": 1,
            }
          }
          type="button"
        >
          June 2019
        </button>
        <button
          aria-label=""
          className="react-calendar__navigation__arrow react-calendar__navigation__next-button"
          onClick={[Function]}
          type="button"
        >
          ›
        </button>
        <button
          aria-label=""
          className="react-calendar__navigation__arrow react-calendar__navigation__next2-button"
          onClick={[Function]}
          type="button"
        >
          »
        </button>
      </div>
      <div
        className="react-calendar__month-view "
      >
        <div
          style={
            Object {
              "alignItems": "flex-end",
              "display": "flex",
            }
          }
        >
          <div
            style={
              Object {
                "flexGrow": 1,
                "width": "100%",
              }
            }
          >
            <div
              className="react-calendar__month-view__weekdays"
              style={
                Object {
                  "display": "flex",
                  "flexDirection": undefined,
                  "flexWrap": "no-wrap",
                }
              }
            >
              <div
                className="react-calendar__month-view__weekdays__weekday"
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
              >
                <abbr
                  aria-label="Monday"
                  title="Monday"
                >
                  Mon
                </abbr>
              </div>
              <div
                className="react-calendar__month-view__weekdays__weekday"
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
              >
                <abbr
                  aria-label="Tuesday"
                  title="Tuesday"
                >
                  Tue
                </abbr>
              </div>
              <div
                className="react-calendar__month-view__weekdays__weekday"
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
              >
                <abbr
                  aria-label="Wednesday"
                  title="Wednesday"
                >
                  Wed
                </abbr>
              </div>
              <div
                className="react-calendar__month-view__weekdays__weekday"
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
              >
                <abbr
                  aria-label="Thursday"
                  title="Thursday"
                >
                  Thu
                </abbr>
              </div>
              <div
                className="react-calendar__month-view__weekdays__weekday"
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
              >
                <abbr
                  aria-label="Friday"
                  title="Friday"
                >
                  Fri
                </abbr>
              </div>
              <div
                className="react-calendar__month-view__weekdays__weekday"
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
              >
                <abbr
                  aria-label="Saturday"
                  title="Saturday"
                >
                  Sat
                </abbr>
              </div>
              <div
                className="react-calendar__month-view__weekdays__weekday"
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
              >
                <abbr
                  aria-label="Sunday"
                  title="Sunday"
                >
                  Sun
                </abbr>
              </div>
            </div>
            <div
              className="react-calendar__month-view__days"
              style={
                Object {
                  "display": "flex",
                  "flexDirection": undefined,
                  "flexWrap": "wrap",
                }
              }
            >
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="May 27, 2019"
                >
                  27
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="May 28, 2019"
                >
                  28
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="May 29, 2019"
                >
                  29
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="May 30, 2019"
                >
                  30
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="May 31, 2019"
                >
                  31
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 1, 2019"
                >
                  1
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 2, 2019"
                >
                  2
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__tile--now react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 3, 2019"
                >
                  3
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 4, 2019"
                >
                  4
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 5, 2019"
                >
                  5
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 6, 2019"
                >
                  6
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 7, 2019"
                >
                  7
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 8, 2019"
                >
                  8
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 9, 2019"
                >
                  9
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 10, 2019"
                >
                  10
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 11, 2019"
                >
                  11
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 12, 2019"
                >
                  12
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 13, 2019"
                >
                  13
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 14, 2019"
                >
                  14
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 15, 2019"
                >
                  15
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 16, 2019"
                >
                  16
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 17, 2019"
                >
                  17
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 18, 2019"
                >
                  18
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 19, 2019"
                >
                  19
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 20, 2019"
                >
                  20
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 21, 2019"
                >
                  21
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 22, 2019"
                >
                  22
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 23, 2019"
                >
                  23
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 24, 2019"
                >
                  24
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 25, 2019"
                >
                  25
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 26, 2019"
                >
                  26
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 27, 2019"
                >
                  27
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 28, 2019"
                >
                  28
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 29, 2019"
                >
                  29
                </abbr>
              </button>
              <button
                className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                onClick={[Function]}
                onFocus={null}
                onMouseOver={null}
                style={
                  Object {
                    "flexBasis": "14.285714285714286%",
                    "marginLeft": null,
                    "maxWidth": "14.285714285714286%",
                    "overflow": "hidden",
                  }
                }
                type="button"
              >
                <abbr
                  aria-label="June 30, 2019"
                >
                  30
                </abbr>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="App-link"
    >
      <div
        className="css-text-901oao"
        dir="auto"
      >
        Hello textasfdsa
      </div>
    </div>
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "backgroundColor": "rgba(255,255,255,1.00)",
          "marginTop": "5px",
        }
      }
    >
      <input
        autoCapitalize="sentences"
        autoComplete="on"
        autoCorrect="on"
        className="css-textinput-1cwyjr8"
        data-focusable={true}
        dir="auto"
        label="First name"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyPress={[Function]}
        onSelect={[Function]}
        placeholder="I am ready!"
        readOnly={false}
        spellCheck={true}
        type="text"
        value=""
      />
      <div
        className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        data-focusable={true}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "rgba(255,0,0,1.00)",
            "marginTop": "5px",
          }
        }
        tabIndex="0"
      >
        <div
          className="css-text-901oao"
          dir="auto"
        >
          Submit
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignItems": "center",
        "WebkitBoxAlign": "center",
        "WebkitBoxPack": "center",
        "WebkitJustifyContent": "center",
        "alignItems": "center",
        "backgroundColor": "rgba(0,110,173,1.00)",
        "height": "200px",
        "justifyContent": "center",
        "marginTop": "10px",
        "msFlexAlign": "center",
        "msFlexPack": "center",
      }
    }
  >
    <div
      className="css-text-901oao"
      dir="auto"
      style={
        Object {
          "fontSize": "20px",
          "fontWeight": "bold",
        }
      }
    >
      This Footer
    </div>
  </div>
</div>
`;
